Doxyfile.in

```diff
@@ -618,7 +618,7 @@ CITE_BIB_FILES         =
 # The QUIET tag can be used to turn on/off the messages that are generated
 # by doxygen. Possible values are YES and NO. If left blank NO is used.
 
-QUIET                  = NO
+QUIET                  = YES
```

Comment:
Plus doxygen plays nicely and outputs to stdout and stderr, so you can just redirect stdout to /dev/null to see the errors.